universal-login-example/src/components/ApproveConnection.js

```diff
@@ -1,32 +1,51 @@
 import ApproveConnectionView from '../views/ApproveConnectionView';
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import {Wallet} from 'ethers';
+import { Wallet } from 'ethers';
```

Comment:
We should probably agree on guidelines regarding spaces in `{}` and put it to linter, otherwise, it generates noise in the repository.

Reply:
Sorry, it's my auto formatter